site/docs/plugins/keyboard.md

```diff
@@ -11,13 +11,14 @@ Let us try to clear it up a bit:
 | ---------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------- |
 | [**Inline Keyboard**](#inline-keyboards) | a set of buttons that is displayed underneath a message inside the chat                                                            |
 | [**Custom Keyboard**](#custom-keyboards) | a set of buttons that is displayed instead of the user's system keyboard                                                           |
-| **Inline keyboard button**               | a button in an inline keyboard, sends a callback query not visible to the user when pressed, sometimes just called _inline button_ |
+| **Inline Keyboard button**               | a button in an inline keyboard, sends a callback query not visible to the user when pressed, sometimes just called _inline button_ |
 | **Custom Keyboard button**               | a button in a keyboard, sends a text message with its label when pressed, sometimes just called _keyboard button_                  |
 | **`InlineKeyboard`**                     | class in grammY to create inline keyboards                                                                                         |
 | **`Keyboard` (!)**                       | class in grammY to create custom keyboards                                                                                         |
 
 > Note that both custom keyboard buttons and inline keyboard buttons can also have other functions, such as requesting the user's location, opening a website, and so on.
 > This was omitted for brevity.
+> As of current tests, it's not possible to edit a custom keyboard with `editMessageReplyMarkup` and change it to an inline keyboard and vice versa.
```

Comment:
```suggestion

It is not possible to specify both a custom keyboard and an inline keyboard in the same message.
The two are mutually exclusive.
Moreover, the sent kind of reply markup cannot be changed at a later point by editing the message.
For example, it is not possible to first send a custom keyboard along with a message, and then edit the message to use an inline keyboard.
```

Reply:
This looks fine. I wanted to make it a small note.

Reply:
I'm OK.

Reply:
Nice, go ahead